apps/docs/src/guide/creating-a-fuel-dapp/deploying-a-dapp-to-testnet.md

```diff
@@ -82,5 +82,5 @@ Congratulations! You have successfully deployed your Fuel dApp to the testnet.
 To recap, to deploy your dApp to the testnet, you need to:
 
 1. Deploy your contract to the testnet using `forc deploy --testnet`.
-2. Specify this contract ID in your frontend code in `src/routes/index.lazy.tsx`.
+2. Specify this contract ID in your frontend's environment variables.
```

Comment:
Shall we say which env var to set?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3178/commits/12a6491e0766b2d83ac6a8ec83a358d75c1d5a09